### ISSUE TITLE
Still active?

### ISSUE THREAD

**Issue by LincMitch** [blue-button/bbClear#10]
Hi,

I've been thinking along the lines of "Bootstrap for Health" and stumbled across this front-end framework.

I believe the concept  has so much potential and was wondering if this repo is still active?
Or maybe a link to another thats out there.

Cheers
Link

**Comment by blacktm** [comment]
Hi @LincMitch. No, not still active, although hopefully it inspired some ideas that still live on. Check out @rypan's [TEDMED talk](http://www.tedmed.com/talks/show?id=91690) which provides background and motivations for this framework.

We wanted to improve how health information was presented to patients, families, and caregivers, and so the "Bootstrap for health records" was definitely the idea here. Important for us was to not build stuff in the abstract, but have a real application in mind. We intended to deploy and test with our Dept. of Veterans Affairs [health portal](https://www.myhealth.va.gov), although unfortunately it was still in the backlog when I left (the new [Vets.gov team](https://www.vets.gov/playbook) has been doing a major digital revamp of the Dept, but health is just a part of the overall effort).

So, my advice for you is to first have a use case / application, build something that works for your org and patients, _then_ consider open sourcing. And chances are, because everyones needs are so disparate, other health orgs or companies won't be able to easily pick it up and use — I think that's the real reason we don't see design frameworks for health information and data out there, although others might have different opinions.

Whatever you choose to do, good luck! And happy to provide more context, ideas, counsel if helpful.

**Comment by LincMitch** [comment]
Thanks Tom for the quick response.

I think your assessment “everyones needs are so disparate, other health orgs or companies won't be able to easily pick it up and use” is a fair call.
So, as apposed to building something that works, I am thinking more of a Design System. A starting point for the discovery / design / prototyping part of the process. I call it Design Code.
I’m looking at using https://github.com/airbnb/react-sketchapp which renders to Native, Web and SketchApp.


Cheers
Link

**Comment by blacktm** [comment]
Yes, I think a design system, principles, will be a more productive place to start. And as you validate ideas, approaches, you might find contained bits to package up and open source (rather than the monolithic approach we took). Looking forward to seeing what you all come up with!

**Comment by stfnh** [comment]
I used this project as a starting point for a [SMART on FHIR](https://smarthealthit.org/) example app. Feel free to fork or contribute! It's just a sample app which I used to get started with SMART on FHIR, but you might find it helpful:

See https://github.com/stfnh/bb-clear-smart-fhir

**Comment by blacktm** [comment]
@stfnh Nice work, thanks for sharing!

**Comment by mak1e** [comment]
Hi @blacktm and @LincMitch would you guys know if the showcase designs can be yse as a reference? I am working on developing a platform for people with diabetes, similar to Lark, Omada Health and Livongo but specific to New Zealand and Australia health standards and govt regulation.

@stfnh nice work, how are you progressing with your Smart on fhir (https://smarthealthit.org/) project? 

@LincMitch I like your idea on “bootstrap for health” there’s quite a few but there’s no such standard? Maybe we could start something where people doing health innovation could vote for?

Looking forward to hearing from you all!

**Comment by stfnh** [comment]
Hi @mak1e 
I used SMART on FHIR to integrate environmental heatlh data with patient records, here is a proof of concept repo: https://github.com/CDCgov/environmental-health-smart-fhir 
I switched jobs, so I am not actively mainting this repo anymore. Good luck with your project, it sounds very relevant! 👨‍⚕
